src/govuk/settings/_measurements.scss

```diff
@@ -103,3 +103,27 @@ $govuk-focus-width: 3px !default;
 /// @access public
 
 $govuk-hover-width: 10px !default;
+
+/// Link underline thickness
```

Comment:
Is measurements the right place for these settings? I also considered creating a new `settings/_links.scss` file for them.

Reply:
Worth thinking about the effect this has on the Sass docs as well – at the minute it'd be under [measurements](https://frontend.design-system.service.gov.uk/sass-api-reference/#measurements) which doesn't feel right. Equally if we did have a `settings/links` it feels a little odd that all of the link colour settings will still be under `settings/colours`.